src/disclosures/css/cf-enhancements.less

```diff
@@ -319,7 +319,7 @@ a.btn__full-small.btn__link {
                  unit( 6px / @base-font-size-px, em );
         border: 0;
         outline: 2px solid transparent;
-        .webfont-regular();
```

Comment:
This was updated to `.u-webfont-regular()`. However, it has since been [deprecated](https://github.com/cfpb/capital-framework/blob/master/src/cf-core/usage.md#webfont-mixins). You may still want to use it though in place of setting the font value directly, so that it's easier to find and strip out in the future.

Reply:
I updated these to variables - thanks for the note!

Reply:
Cool, thanks!